apps/backend/src/app/api/latest/payments/purchases/verification-code-handler.tsx

```diff
@@ -0,0 +1,19 @@
+import { createVerificationCodeHandler } from "@/route-handlers/verification-code-handler";
+import { VerificationCodeType } from "@prisma/client";
+import { offerSchema, yupObject, yupString } from "@stackframe/stack-shared/dist/schema-fields";
+
+export const purchaseUrlVerificationCodeHandler = createVerificationCodeHandler({
+  type: VerificationCodeType.PURCHASE_URL,
+  method: yupObject({}),
+  data: yupObject({
+    tenancyId: yupString().defined(),
+    customerId: yupString().defined(),
+    offer: offerSchema,
+    stripeCustomerId: yupString().defined(),
+    stripeAccountId: yupString().defined(),
+  }),
+  // @ts-ignore TODO: fix this
+  async handler(_, __, data) {
+    return null;
+  },
```

Comment:
What is the issue here? would throwing an error work?

```suggestion
  async handler(_, __, data) {
    throw new StackAssertionError("Verification code handler called, but never used?");
  },
```